### DISCUSSION TITLE
VGGish training

### DISCUSSION THREAD

**Discussion by PRamoneda** [MTG/essentia#1126] [Essentia Models]
Hi @palonso!

I don't understand where do you start to train the VGGish after the transfer learning

Thanks!

**Comment by palonso** [comment]
@PRamoneda,
in the transfer learning classifiers we train two dense layers appended to a frozen VGGish architecture with the weights from the [original implementation](https://github.com/tensorflow/models/tree/master/research/audioset/vggish).
